webpack/ForemanInventoryUpload/Components/AccountList/AccountListSelectors.js

```diff
@@ -6,3 +6,5 @@ export const selectAccounts = state => selectAccountsList(state).accounts;
 export const selectPollingProcessID = state =>
   selectAccountsList(state).pollingProcessID;
 export const selectError = state => selectAccountsList(state).error;
+export const selectAutoUploadEnabled = state =>
+  selectAccountsList(state).autoUploadEnabled || false;
```

Comment:
it's only to ensure that when we receive `undefined` it will be `false`, for example when the state doesn't exist yet.
but maybe you are right, maybe we should leave it as it is, 
and in case of `undefined`, `proptypes` will already take care of it with its default value which will be `true`